base/reflection.jl

```diff
@@ -159,17 +160,23 @@ Determine the module containing the definition of a `DataType`.
 """
 datatype_module(t::DataType) = t.name.module
 
-isconst(s::Symbol) = ccall(:jl_is_const, Cint, (Ptr{Void}, Any), C_NULL, s) != 0
-
 """
-    isconst([m::Module], s::Symbol) -> Bool
+    isconst(m::Module, s::Symbol) -> Bool
 
-Determine whether a global is declared `const` in a given `Module`. The default `Module`
-argument is [`current_module()`](@ref).
+Determine whether a global is declared `const` in a given `Module`.
 """
 isconst(m::Module, s::Symbol) =
     ccall(:jl_is_const, Cint, (Any, Any), m, s) != 0
 
+"""
+    @isdefined s -> Bool
+
+Get whether symbol `s` is defined in the current scope.
```

Comment:
"Get whether" sounds a bit strange, that wording isn't used in any other docstrings